### ISSUE TITLE
Feature Request: Tab close functionalities

### ISSUE THREAD

**Issue by akoenig** [graphcool/graphql-playground#596]
#### This issue pertains to the following package(s):

- [x] GraphQL Playground - Electron App
- [x] GraphQL Playground HTML
- [x] GraphQL Playground

#### Feature Request

Would be awesome to have a little context menu on a tab with the following functionalities:

  * Close Others
  * Close to the Right

**Comment by playerx** [comment]
Yep, when there are lots of tabs (and there are always lots of tabs) it will be really handy, I was going to create same issue

**Comment by sagar-tts** [comment]
Is that possible to send request after close the window tab.

I want to implement after close tab and send graphql request for change user status.

Please help for this.

Thanks,
